seaborn/categorical.py

```diff
@@ -1942,41 +1936,47 @@ def horz_perc_box(x, b, i, k, w):
             hex_color = mpl.colors.rgb2hex(color)
 
             if vert:
-                boxes = [vert_perc_box(x, b[0], i, k, b[1])
-                         for i, b in enumerate(zip(box_ends, w_area))]
+                box_func = vert_perc_box
+                xs_median = [x - widths / 2, x + widths / 2]
+                ys_median = [y, y]
+                xs_outliers = np.repeat(x, len(outliers))
```

Comment:
Not a big deal but `np.full(len(outliers), x)` might be more idiomatic.

Reply:
Thanks, will change this.